migration/db/migrations/20221103182052_add_user_table.sql

```diff
@@ -0,0 +1,48 @@
+-- +micrate Up
+-- SQL in section 'Up' is executed when this migration is applied
+
+-- Table for model PlaceOS::Model::User
+CREATE TABLE IF NOT EXISTS "user"(
+     created_at TIMESTAMPTZ NOT NULL,
+   updated_at TIMESTAMPTZ NOT NULL,
+   name TEXT NOT NULL,
+   nickname TEXT NOT NULL,
+   email JSONB NOT NULL,
```

Comment:
should the email column be JSON type?
I assume it should be TEXT

Reply:
yeah I think for frontend compatibility we should make it a text column
currently that Email class just implements one attribute: `address`

Reply:
Yeah I think that's ideal and how it was handled previously
It was a little obscure how the old version handled the field, basically the Email class acted like a string for json serialisation and deserialisation so was transparent to the database as it stored things as JSON